src/templates/program-page.js

```diff
@@ -717,14 +703,10 @@ export const query = graphql`query ($id: String) {
                           field_link_image {
                             relationships {
                               field_media_image {
-                                localFile {
-                                  publicURL
-                                  childImageSharp {
-                                    resize(width: 400, height: 300, cropFocus: CENTER) {
-                                      src
-                                    }
-                                  }
-                                }
+                                gatsbyImage( 
+                                  cropFocus: CENTER
```

Comment:
Assuming publicURL is not in use and we're getting rid of the height because it's unnecessary?